app-text/podofo/podofo-0.9.5.recipe

```diff
@@ -81,6 +81,7 @@ BUILD_PREREQUIRES="
 	cmd:cmake
 	cmd:make
 	cmd:sed
+	devel:libcppunit$secondaryArchSuffix
```

Comment:
I might be missing something, but I'm quite sure `devel:libcppunit$secondaryArchSuffix` should be in BUILD_REQUIRES, not in BUILD_**PRE**REQUIRES.